src/Writer/Extension/PodcastIndex/Entry.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-feed for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-feed/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-feed/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\Feed\Writer\Extension\PodcastIndex;
+
+use Laminas\Feed\Writer;
+use Laminas\Stdlib\StringUtils;
+use Laminas\Stdlib\StringWrapper\StringWrapperInterface;
+
+class Entry
+{
+    /**
+     * Array of Feed data for rendering by Extension's renderers
+     *
+     * @var array
+     */
+    protected $data = [];
+
+    /**
+     * Encoding of all text values
+     *
+     * @var string
+     */
+    protected $encoding = 'UTF-8';
+
+    /**
+     * The used string wrapper supporting encoding
+     *
+     * @var StringWrapperInterface
+     */
+    protected $stringWrapper;
+
+    public function __construct()
+    {
+        $this->stringWrapper = StringUtils::getWrapper($this->encoding);
+    }
+
+    /**
+     * Set feed encoding
+     *
+     * @param  string $enc
+     * @return $this
+     */
+    public function setEncoding($enc)
+    {
+        $this->stringWrapper = StringUtils::getWrapper($enc);
+        $this->encoding      = $enc;
+        return $this;
+    }
```

Comment:
```suggestion
    /**
     * Set feed encoding
     */
    public function setEncoding(string $encoding): Entry
    {
        $this->stringWrapper = StringUtils::getWrapper($encoding);
        $this->encoding      = $encoding;
        
        return $this;
    }
```

Reply:
I would recommend using the `:self` notation